src/graphql/search/cms/filters.ts

```diff
@@ -1,38 +1,18 @@
-import fetch from 'node-fetch'
-import { formatThemeFilters, formatDateFilters } from './normalize'
-import withCache from '../../utils/withCache'
+import { JsonAPI, formatThemeFilters, formatDateFilters } from './normalize'
 import { Filter } from '../../../generated/graphql'
 
-interface FilterCount {
-  theme: Array<{
-    key: string
-    count: number
-  }>
+type FilterCount = {
+  theme: Array<ThemeFilterCount>
 }
 
-const week = 60 * 60 * 24 * 7
-
-export const themeTaxonomyCached = async () =>
-  await withCache(
-    'themeTaxonomy',
-    () => fetch(`${process.env.CMS_URL}/jsonapi/taxonomy_term/themes`).then(res => res.json()),
-    week,
-  )
-
-export default async (filterCount: FilterCount): Promise<any> => {
-  let filters
-  try {
-    const themeTaxonomy: any = await Promise.resolve(themeTaxonomyCached())
-    const themeFilters: Filter = formatThemeFilters(themeTaxonomy, filterCount.theme)
-
-    const dateFilters: Filter = formatDateFilters()
+export type ThemeFilterCount = {
+  key: number
+  count: number
+}
 
-    filters = [themeFilters, dateFilters]
-  } catch (e) {
-    console.warn(e)
-  }
+export default (filters: JsonAPI, filterCount?: FilterCount): Array<Filter> => {
```

Comment:
No need to specify a return type here as it can be inferred from the returned value.